mobile/App.tsx

```diff
@@ -71,6 +71,7 @@ const App = () => {
     loadCookie('settings_mode');
     loadCookie('settings_light_qr');
     loadCookie('settings_network');
+    loadCookie('settings_use_proxy');
```

Comment:
I don't like this solution, it took me 30 min to realize if you don't do this the cookie won't work

Reply:
Yes! lol. I had this same issue long time ago when I added the network and light_qr settings... Did not know why it was not working :facepalm: